packages/components/organisms/f-checkout/src/components/UserNote.vue

```diff
@@ -39,11 +39,11 @@ $userNote-textArea-borderWidth                   : 1px;
 $userNote-textArea-borderColour                  : $grey--light;
 $userNote-textColour                             : $color-text;
 $userNote-textArea-bg                            : $color-bg--component;
-$userNote-textArea-height                        : 144px;
+$userNote-textArea-height                        : 188px;
```

Comment:
I'd personally have liked to remove the duplicated copy rather than increasing the height here but have been told to keep the experience consistent across devices.